bioptim/dynamics/configure_problem.py

```diff
@@ -163,6 +163,8 @@ def torque_driven(
                 implicit_dynamics=implicit_dynamics,
             )
 
+        ConfigureProblem.configure_inverse_dynamics_function(ocp, nlp)
```

Comment:
Does this make sense if `with_contact`? Please make sure to add the proper raises in the configure function

Reply:
Revised